app/serializers/revenue_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class RevenueSerializer < ActiveModel::Serializer
+  attributes :revenue
+
+  def revenue
+    '%.2f' % object
+  end
```

Comment:
**Found this solution on StackOverflow to format number to always have 2 decimals**